sdk/js/src/StreamingConnection.ts

```diff
@@ -17,7 +17,7 @@ export class StreamingConnection {
             this.onMessage(event.data)
         }
         this.connection.onerror = (err) => {
-            this.logger.error('StreamingConnection error', err)
+            this.logger.warn(`StreamingConnection error ${err}`)
```

Comment:
maybe instead of saying error, say something like "StreamingConnection interrupted, re-establishing.."

Reply:
also according to MDN this situation actually means the connection failed to be established in the first place, not that it was closed later. Are we sure this isn't a problem?
https://developer.mozilla.org/en-US/docs/Web/API/EventSource/error_event